exercises/tower-of-hanoi/hanoi.go

```diff
@@ -0,0 +1,30 @@
+package towerofhanoi
+
+// This is an stub file. Remove this comment and
+// implement the function Solve below.
+
+// Solve returns list of moves for solving the Tower of Hanoi puzzle.
+//
+// 'disks' is the amount of disks.
+// 'from' is the id for the source rod.
+// 'to' is the id for the destination rod.
+// 'via' is the id for the helper rod.
+//
+// A move is a [2]int value specifying source and destination rods.
```

Comment:
This strikes me as a little weird. I've literally never used an array in Go. Would a struct `{from, to int}` be a bit more idiomatic?

Reply:
Please see https://github.com/exercism/go/pull/1272/commits/1741aab3426a00e8173927631eeed43300ab8b13